ui/src/app/workflows/components/workflow-labels/workflow-labels.tsx

```diff
@@ -0,0 +1,63 @@
+import * as React from 'react';
+import * as models from '../../../../models';
+
+require('./workflow-labels.scss');
+
+interface WorkflowLabelsProps {
+    workflow: models.Workflow;
+    onChange: (key: string) => void;
+}
+
+export class WorkflowLabels extends React.Component<WorkflowLabelsProps, {hidden: boolean}> {
+    constructor(props: WorkflowLabelsProps) {
+        super(props);
+        this.state = {hidden: true};
+    }
+
+    public render() {
+        const labels = [];
+        if (!this.state.hidden) {
+            const w = this.props.workflow;
```

Comment:
```suggestion
            const wf = this.props.workflow;
```
`wf` is a standard variable name for Workflow